src/renderer/components/reader/ReaderMenu.tsx

```diff
@@ -195,9 +219,55 @@ export class ReaderMenu extends React.Component<Props, State> {
         return undefined;
     }
 
+    private buildGoToPageSection() {
+        const { __ } = this.props;
+        const error = this.state.pageError;
+        return <div className={styles.goToPage}>
+            <p className={styles.title}>{__("reader.navigation.goToTitle")}</p>
+            {error &&
+                <p
+                    className={styles.goToErrorMessage}
+                    aria-live="assertive"
+                    aria-relevant="all"
+                    role="alert"
+                >
+                    { __("reader.navigation.goToError") }
+                </p>
+            }
```

Comment:
I think that the error should be located below the text input field, because right now the error "pushes down" the interactive input control when it appears.